vuln/npm/399.json

```diff
@@ -0,0 +1,21 @@
+{
+  "id": "399",
```

Comment:
Nit: for most entries, except for the three of 390, 387, 389, this is a number and not a string.
I suggest to use a number here.

Reply:
yeah good catch, will update!